### PR TITLE
Only allow portage tests to run on gentoo systems

### PR DESCRIPTION
**Pull request by xeacott**
### What does this PR do?
Prevents Windows from running portage tests. Windows does not support this software there is not a need for this test to run.

### What issues does this PR fix or reference?
Fixes:
PR Test Failure on Windows from this comment: https://github.com/saltstack/salt/pull/60306#issuecomment-869806087

### Previous Behavior
Consistently fails on Windows because it is looking for specific gentoo/funtoo software

### New Behavior
Skips this test on systems where portage is not supported

### Merge requirements satisfied?
**[NOTICE] Bug fixes or features added to Salt require tests.**

- [ ] Docs
- [ ] Changelog - https://docs.saltstack.com/en/master/topics/development/changelog.html
- [ ] Tests written/updated

### Commits signed with GPG?
Yes/No

Please review [Salt's Contributing Guide](https://docs.saltstack.com/en/master/topics/development/contributing.html) for best practices.

See GitHub's [page on GPG signing](https://help.github.com/articles/signing-commits-using-gpg/) for more information about signing commits with GPG.

### CODE REVIEW

**Comment by marbx** [comment]
Oh good: "Skip" instead of  "DummyAtom" makes the code shorter and more expressive.

**Review by waynew — DISMISSED** [review]

File: tests/unit/modules/test_portage_config.py [hunk 1]
```diff
@@ -1,65 +1,35 @@
...
 import salt.utils.files
 from tests.support.mixins import LoaderModuleMockMixin
-from tests.support.mock import MagicMock, patch
-
-# Import Salt Testing libs
+from tests.support.mock import patch
 from tests.support.runtests import RUNTIME_VARS
-from tests.support.unit import TestCase
+from tests.support.unit import TestCase, skipIf
+
```
Comment (s0undt3ch) [inline tests/unit/modules/test_portage_config.py:14]: `pytest.importorskip("portage")` and be done with changes....

:)

**Review by s0undt3ch — CHANGES_REQUESTED** [review]

**Comment by s0undt3ch** [comment]
Can I stretch the repo as and ask you to migrate the test module to pytest?

**Comment by xeacott** [comment]
> Can I stretch the rope as and ask you to migrate the test module to pytest?

Absolutely not. Just kidding. Yeah i'll migrate this over!

File: tests/unit/modules/test_portage_config.py (hunk 1 above)
Comment (xeacott) [reply tests/unit/modules/test_portage_config.py:14]: Cute. Ya learn something new everyday

**Comment by marbx** [comment]
@xeacott, why have you moved the file and not `git mv` 'ed?

**Review by garethgreenaway — DISMISSED** [review]

**Review by garethgreenaway — APPROVED** [review]

**Pull request merged**